RemoteryConfig.cmake.in

```diff
@@ -0,0 +1,47 @@
+# Remotery cmake module
+#
+# This module defines the following variables in your project::
+#
+#   Remotery_FOUND          - True if remotery is available
+#   Remotery_INCLUDE_DIR    - Include directories for remotery
+#   Remotery_LIBRARY        - The library for dynamically linking
+#   Remotery_STATIC_LIBRARY - The library for statically linking
```

Comment:
For now, `Remotery_FOUND` can be resolved in downstream projects but `Remotery_INCLUDE_DIR`, `Remotery_LIBRARY`, and `Remotery_STATIC_LIBRARY` cannot.

I am trying to understand why.

Generally, might it be easier to use `remotery` rather than `Remotery` as a stub everywhere. To me, it would be more in line with canonical lower-cased names; this might also help prevent misconfiguration and errors.

Do you think is it sensible as a naming? Do you think it is possible?